app/controllers/orders_controller.rb

```diff
@@ -25,7 +25,11 @@ def show
   private
 
   def order_params
-    params.require(:order).permit(:shipping_name, :product_id, :zipcode, :address).merge(paid: false)
+    params.require(:order).permit(:product_id).merge(paid: false, shipping_address: Address.new(address_params))
```

Comment:
switched to using nested attributes in https://github.com/ams11/lovevery-homework/pull/3